modules/apps/portal-vulcan/portal-vulcan-api/build.gradle

```diff
@@ -10,7 +10,9 @@ dependencies {
 	compileOnly group: "javax.ws.rs", name: "javax.ws.rs-api", version: "2.1"
 	compileOnly group: "org.apache.cxf", name: "cxf-core", version: "3.2.14"
 	compileOnly group: "org.apache.felix", name: "org.apache.felix.http.servlet-api", version: "1.1.2"
+	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
```

Comment:
Needing the OSGi component annotation in Vulcan API seems like a smell to me... we have a component in the Vulcan API module?

Reply:
Mmmm, that should not be there. I will review that. Thanks :)